### DISCUSSION TITLE
为什么安装教程里webrtcvad-wheels 要单独安装

### DISCUSSION THREAD

**Discussion by wei-z-git** [babysor/MockingBird#718] [Q&A] [answered]
rt, 如何判断我是否需要这个包呢
https://github.com/babysor/MockingBird/blob/main/README-CN.md#1-%E5%AE%89%E8%A3%85%E8%A6%81%E6%B1%82

**Comment by babysor** [comment, marked as answer]
早期安装的时候，如果指定版本，容易有不兼容的结果，现在如果稳定版本能兼容mac和windows，也非常欢迎尝试加到requirement.txt 来减少操作步骤

**Comment by wei-z-git** [comment]
windows+python3.9.12+webrtcvad-2.0.1可用
